package.json

```diff
@@ -55,6 +55,7 @@
     "moment": "^2.15.1",
     "reflect-metadata": "^0.1.8",
     "rxjs": "^5.4.0",
+    "chokidar": "3.5.2",
```

Comment:
is this 3.5.2 version of `chokidar` needed? `chokidar` version 2.1.8 is still listed in the package-lock.json since `glob-watcher` has a dependency on it, and `chokidar` 2.1.8 has a dependency on `"fsevents": ^1.2.7`. The patched version of `fsevents` is 1.2.11, so I would expect it to resolve to an updated version if you regenerate the package-lock.json.